Visual Studio/Realm.vcxproj

```diff
@@ -221,7 +221,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Static library, release|x64'" Label="Configuration">
     <ConfigurationType>StaticLibrary</ConfigurationType>
-    <CharacterSet>Unicode</CharacterSet>
+    <CharacterSet>MultiByte</CharacterSet>

```

Comment:
This is required because UWP only has the w_char API available (the API methods that begin with W) and no longer the legacy ANSI versions (those that begin with A)